scripts/release.mjs

```diff
@@ -60,7 +60,7 @@ async function build() {
   await $({
     stdout: 'inherit',
     stderr: 'inherit',
-  })`sentry-cli releases set-commits --auto ${nextVersion}`;
+  })`sentry-cli releases set-commits --auto ${nextVersion} --ignore-missing`;
```

Comment:
I rebased at some point which broke the `sentry-cli` because it couldn't find the commits. Passing `--ignore-missing` allows the CLI to fallback to the default behavior if it hits this error. Based on their [docs](https://docs.sentry.io/cli/releases/#dealing-with-missing-commits), passing this every time shouldn't have a downside.

Reply:
Send it!